app/modules/topics/tests/reducer.test.js

```diff
@@ -22,20 +20,6 @@ describe(`reducer`, (): void => {
     description: '',
     rootContentItemId: 'abcdefghij',
   };
-  const dummyInitialState = {
-    byId: dummyTopicsById,
-  };
-
-  it(`returns the initial state, when state parameter is undefined`, (): void => {
-    const dummyAction = {
-      type: t.ADD_TO_STATE_ERROR,
-      error: {
-        message: `Flow will complain if the passed action isn't some kind of valid TopicAction.`,
-      },
-    };
-
-    expect(reducer(undefined, dummyAction)).toEqual(dummyInitialState);
-  });
```

Comment:
Why did you delete this? Returning the initial state when the passed state is undefined is part of the redux spec; all reducers should have a test for this. Just use a different kind of dummy TopicAction instead.